modules/apps/jaggery/emm/src/emm/modules/policy.js

```diff
@@ -31,6 +31,63 @@ policyModule = function () {
     var publicMethods = {};
     var privateMethods = {};
 
+    privateMethods.getAllPoliciesOnError = function (responsePayload) {
+        var response = {};
+        response.status = "error";
+        /* the status message used other than the status code because the status code return from IS for different
+         occasions with under status messages. */
+        if (responsePayload == "Scope validation failed") {
+            response.content = "Permission Denied";
+        } else {
+            response.content = responsePayload;
+        }
+        return response;
+    };
+
+    privateMethods.getAllPoliciesOnSuccess = function (responsePayload) {
```

Comment:
function name does not convey a proper meaning upon the actual usage of this function